real_estate/real_estate/doctype/property_booking_order/property_booking_order.py

```diff
@@ -206,6 +218,34 @@ def get_installment_date(payment_plan, installment):
 
 	return start_date
 
+@frappe.whitelist()
+def get_customer_details(customer):
+	out = frappe._dict()
+
+	customer_details = frappe.get_doc('Customer', customer)
+	# Customer Name
+	out.customer_name = customer_details.customer_name
+	out.mobile_no = customer_details.mobile_no
```

Comment:
Contact details like number and email should not come from Customer master. It should come from the selected `Contact`